moveit_ros/moveit_servo/test/servo_launch_test_common.hpp

```diff
@@ -454,10 +467,21 @@ class ServoFixture : public ::testing::Test
 protected:
   rclcpp::Node::SharedPtr node_;
   rclcpp::Executor::SharedPtr executor_;
-  moveit_servo::ServoParametersPtr parameters_;
   std::thread executor_thread_;
+  std::shared_ptr<const moveit_servo::ServoParameters> servo_parameters_;
+  bool servo_parameters_loaded_success_ = false;
+
+  struct TestParameters
+  {
+    double PUBLISH_HZ;
```

Comment:
The ROS2 style guide refers to the Google style guide here. It says the variable names should be camel_case.

https://google.github.io/styleguide/cppguide.html#Variable_Names

Reply:
You are right.  I named these this way to make it clear that they are constants everywhere they are used.  I agree though that this is not following the style guide and I will fix it.